backend/bin/bootsnap

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env ruby
```

Comment:
Is it common in ruby to include all of these ./bin/* files in the repo itself? Or is this something that should be ignored and built by contributors?

Reply:
Yeah, it's common as a first-time set up step. I forgot to do it in the `main` branch, so I'm doing it here.

The idea behind generating these "binstubs", as they're called, is to provide a version of several of the bin files you might use while developing a Rails application that are tied to the versions of the gems in your project. In terms of React, imagine if you had some packages you kept version-locked in `package.json`, like `prettier` for example. You'd then run the `prettier` executable in your project directory and it would run the version specified by your `package.json` file.

To make use of these binstubs, all you need to do is prepend `bin/` to whatever you want to do, e.g. `bin/rails`.

I hope that wasn't just a bunch of word soup! I can get a little wordy when explaining things, but the TLDR is: These are executables installed by Rails that are tied to the version of Rails used by this project.